src/services/emailService.js

```diff
@@ -0,0 +1,39 @@
+const nodemailer = require('nodemailer');
+
+require('dotenv').config();
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
```

Comment:
Ensure that the transporter configuration includes the `secure` option if you are using a port that requires SSL/TLS, such as 465. If using port 587 or 25, ensure that the `secure` option is set to `false`.